pkg/api/graph/graph.go

```diff
@@ -508,7 +508,7 @@ func ReverseGraphEdge(g Interface, from, to graph.Node, edgeKinds sets.String) b
 // AddReversedEdge adds a reversed edge for every passed edge and preserves the existing
 // edge. Used to convert a one directional edge into a bidirectional edge, but will
 // create duplicate edges if a bidirectional edge between two nodes already exists.
-func AddReversedEdge(g Interface, from, to graph.Node, edgeKinds sets.String) bool {
+func AddReversedEdge(g Interface, from, to graph.Node) bool {
```

Comment:
> I'll make my own (sane) function if for some reason you absolutely can't budge on this.

So because no one is using now, you'll bork the function so that it can't be used to reverse `SuccessorEdges` or any other function that takes an `EdgeFunc`?  No, I won't budge.  You can't just destroy power.